embeddings/evaluator/submission.py

```diff
@@ -14,19 +17,40 @@
 from embeddings.evaluator.sequence_labeling_evaluator import SequenceLabelingEvaluator
 from embeddings.evaluator.text_classification_evaluator import TextClassificationEvaluator
 from embeddings.utils.json_dict_persister import CustomJsonEncoder
+from embeddings.utils.utils import standardize_name
 
 
 @dataclass
-class Submission:
+class _BaseSubmission(ABC):
```

Comment:
how about setting `from_local_disk` and `from_wandb` as abstract methods?

Reply:
Arguments for both functions are different so mypy does not allow doing it.